src/interface/swarm_container.hpp

```diff
@@ -43,7 +43,8 @@ class SwarmContainer {
   // Public Methods
   //-----------------
   // Constructor does nothing
-  SwarmContainer() {}
+  SwarmContainer() = default;
+  explicit SwarmContainer(const std::string &name) : swarm_name_(name) {}
```

Comment:
Not that I object, but why this change?

Reply:
this was part of refactoring how the `Add` functionality works with `MeshData`.  all container like objects now have names which can be passed into their constructors.